### ISSUE TITLE
#editContentlet fail in edit mode when added it directly to the advanced template

### ISSUE THREAD

**Issue by fmontes** [dotCMS/core#19220]
**Describe the bug**
If you add the macro directly to the template code on in a vtl and dotParse it and try to use it you get
![image.png](https://images.zenhubusercontent.com/570fbdcebf93871ccd997222/54870a9a-e7e8-4f7e-8da4-3d504865e6f3)

**To Reproduce**
1. Create a advanced template and add the following code:
```
#foreach($con in $dotcontent.pull("+contentType:webPageContent +(conhost:48190c8c-42c4-46af-8d1a-0cd5db894797 conhost:SYSTEM_HOST)",10,"modDate desc"))
    #editContentlet($con.inode)
    <h2>$con.title</h2>
    </br>
    </br>
#end
```
2. Create a page and use the created template
3. Edit the page and click the pencil icon to edit
4. See the error

**Expected behavior**
It should edit the piece of content normally

## Acceptance Criteria

1. Edit the piece of content with no errors
2. Work in all the supported browsers (don't forget IE11 and iPad)
3. Multilanguage
4. Unit test

**Comment by erickgonzalez** [comment]
Updated steps for testing. 
Working as it should, there is no error at the browser console and able to edit content without issues.

**Comment by bryanboza** [comment]
Fixed, tested on release-5.3.8 // Postgres // FF now we are able to edit the content without problems
